jack.py

```diff
@@ -674,6 +675,38 @@ def transport_stop(self):
         """Stop JACK transport."""
         _lib.jack_transport_stop(self._ptr)
 
+    def set_timebase_callback(self,
+                              conditional, timebase_callback, userdata=None):
```

Comment:
Please change the order of `conditional` and `timebase_callback` and make former optional with `conditional=False` (I guess this is the right default value?).

Please also rename `timebase_callback` to `callback`, like in the other callback setters.